LearnosityDemo/Pages/ItemsAPIDemo.cshtml

```diff
@@ -2,12 +2,11 @@
 @model LearnosityDemo.Pages.ItemsAPIDemoModel
 @{
     ViewData["Title"] = "Learnosity Example: Standalone Assessment";
-    ViewData["TopJS"] = "<script src=\"https://items.learnosity.com/?v2021.2.LTS\"></script>";
+    ViewData["TopJS"] = "<script src=\"https://items.learnosity.com/?developer\"></script>";
```

Comment:
Maybe we can do something like "latest-lts" here, instead of "developer"? I wanted to emphasise using an LTS version in production.